packages/superset-ui-preset-chart-xy/src/Line/Line.tsx

```diff
@@ -22,9 +22,11 @@ import ChartLegend from '../components/ChartLegend';
 
 chartTheme.gridStyles.stroke = '#f1f3f5';
 
+const DEFAULT_MARGIN = { top: 20, right: 20, left: 20, bottom: 20 };
+
 const defaultProps = {
   className: '',
-  margin: { top: 20, right: 20, left: 20, bottom: 20 },
+  margin: DEFAULT_MARGIN,
```

Comment:
I don't think you need this since you're doing `margin: { ...DEFAULT_MARGIN, ...margin },` down below anyway

Reply:
It is useful for deriving the type as well. (`typeof defaultProps`)